### ISSUE TITLE
Refactor data structures

### ISSUE THREAD

**Issue by Mast3rwaf1z** [aau-p9s/autoscaler-frontend#27]
#26 showed that the data is too unstructured to easily work with. With a refactor like moving the ML portion to a seperate project, and communicating over http using kubernetes' dns, while wrapping forecasts and static forecasts in objects that handle different parts of the project is a better solution than to have Database and Forecaster handle everything.

Additionally moving communication within Kubernetes to its own modules would significantly improve the readability of the code 

Moving the python part into another repo and containerizing that separately is optional as it goes against the design of the project, but I think it's a better solution

In other words: modularize the hell out of this code

**Comment by Mast3rwaf1z** [comment]
By static forecasts i mean the forecasts set manually in the GUI which isn't implemented yet
